src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -61,8 +69,8 @@
 
   @Override
   public int execute(ExecutionContext context) throws IOException, InterruptedException {
-
-    ProcessBuilder processBuilder = new ProcessBuilder(createCommand());
+    ImmutableList<String> command = createCommand();
+    ProcessBuilder processBuilder = new ProcessBuilder(command);
```

Comment:
We totally should, but I'd rather spend that effort on getting Windows CI which would catch this in the first place (`ProcessBuilder` doesn't properly escape the arguments on Windows, so you might be okay, but you might also fail in a non-obvious way.  The other two classes work around that issue.)